packages/TextEditor/src/TextEditor.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function TextEditor() {
+  return <>Text editor</>;
```

Comment:
Is it too early for this PR?

Reply:
trying to make the PR smaller and in stages :)